code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -18,7 +18,7 @@
 	species_traits = list(DYNCOLORS, AGENDER, NO_UNDERWEAR, HAIR, FACEHAIR, HAS_FLESH, HAS_BONE) // i mean i guess they have blood so they can have wounds too
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
 	species_language_holder = /datum/language_holder/ethereal
-	sexes = FALSE //no fetish content allowed
+	sexes = TRUE //Fetish content enabled
```

Comment:
This isn't a modular file so you should mark it with a `SKYRAT EDIT - ORIGINAL: FALSE` 